suite-common/message-system/config/config.v1.json

```diff
@@ -245,14 +245,14 @@
             "conditions": [
                 {
                     "environment": {
-                        "desktop": "<23.7.2",
+                        "desktop": "<23.10.1",
                         "mobile": "!",
-                        "web": "<23.7.2"
+                        "web": "!"
```

Comment:
Is it intentional to exlude web?
When loading the web suite on production url, most clients will refetch the latest from server, but we can't rely on it 100%. Some custom browsers (or PWAs if we support that), may not always refetch, so it is possible to have an outdated client cached.
I'd guesstimate that it will be a very small minority of clients, but keeping the message for them does not cost us anything I think :slightly_smiling_face: 

But ofc idk what is desired..

Reply:
Hmm, that's interesting. @vdovhanych what do you think?